.docker/ascii.art

```diff
@@ -0,0 +1,19 @@
++-------------------------+
+|ooooo:.   .:ooooooooooooo|
+|ooooo.      ::ooooooooooo|
+|ooooo:.       .oooooooooo|
+|ooooooo:.       .:ooooooo|
+|ooooo:...          :ooooo|
+|ooo:.     ..        .:ooo|
+|o:.      :ooo.         :o|
+|o.       .:oooo.        o|
+|.       . ..ooooo..     .|
+|    :oooooooooooooo.     |
+|    .:::::::oooooo:      |
+|.         .:oooo:.      .|
+|o.       :oooo:.       .o|
+|oo.      :o::.        .oo|
+|ooo:..              .oooo|
+|oooooo:..       ..:oooooo|
++-------------------------+
+ Doctrine Mongo ODM Module
```

Comment:
While the module may still be called "DoctrineMongoODMModule", we should start using "MongoDB" as the proper name for the database.

Reply:
Lucky you, it'll still fit ;)